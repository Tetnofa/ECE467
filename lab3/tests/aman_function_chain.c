int func1(int x) {
    return (x + 1);
}

int func2(int x, bool y) {
    if (y) {
        // return (func1(x) * 2);
    }
    return func1(x);
}

bool func3(int x, int y) {
    return (func1(x) > func1(y));
}

void main() {
    int result;
    result = func2(func1(5), func3(3, 2));
    result = func2(func1(5), func1(4));
    writeInt(result);
} 