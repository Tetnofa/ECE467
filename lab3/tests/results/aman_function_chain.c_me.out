sema: 19:13 : no matching definition for func2
sema: 19:4 : type mismatch 
sema: 20:4 : use of undefined identifier writeInt
